students/mathewcm/lesson02/series.py

```diff
@@ -0,0 +1,56 @@
+def fibonacci(n):
+    """
+    This function will return the nth iteration of that value in the fibonacci sequence.
+    """
+    if n == 1:
+        return [1]
+    if n == 2:
+        return [1, 1]
+    # n > 2
+    a = 1
+    b = 1
+    # First two members of the series
+    series = [a, b]
+    for i in range(n):
+        c = a + b
+        series.append(c)
+        a = b
+        b = c
+    return series[-4]
```

Comment:
Why -4? 

You only want to compute as many as you need.

And could you do it without storing the whole list?

Also — this is an iterative vs. a recursive solution— can you do it recursively?